package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"private": true,
 	"scripts": {
-		"ava": "TS_NODE_FILES=true ava",
+		"ava": "rollup test/*.ts --plugin typescript --dir .built/test --chunkFileNames _.mjs --entryFileNames [name].mjs && ava",
```

Comment:
I tried using webpack (too much config) and Parcel (less config) but for one reason or another I couldn't make the tests pass or the build complete successfully.

Thoughts on using `.built/test` as the output folder?

Reply:
> Thoughts on using .built/test as the output folder?

👍